File: scala-pr/54/scala/code-quality-naming/Calculator.scala
class Calculator {
  // {fact rule=code-quality-naming@v1.0 defects=1}
  def calc(a: Double, b: Double, op: String): Double = {
    op match {
      case "+" => a + b
      case "-" => a - b
      case "*" => a * b
      case "/" => if (b != 0) a / b else 0
      case _ => 0
    }
  }
  // {/fact}
}

File: php-pr/51/php/code-quality-naming/Calculator.php
<?php

class Calculator
{
    // {fact rule=code-quality-naming@v1.0 defects=1}
    public function calc($a, $b, $op)
    {
        switch ($op) {
            case '+':
                return $a + $b;
            case '-':
                return $a - $b;
            case '*':
                return $a * $b;
            case '/':
                return $b != 0 ? $a / $b : 0;
            default:
                return 0;
        }
    }
    // {/fact}
}